pkg/controller/database/rdsinstance.go

```diff
@@ -169,11 +170,20 @@ func (e *external) Create(ctx context.Context, mg resource.Managed) (managed.Ext
 		}
 	}
 
-	req := e.client.CreateDBInstanceRequest(rds.GenerateCreateDBInstanceInput(meta.GetExternalName(cr), pw, &cr.Spec.ForProvider))
-	_, err = req.Send(ctx)
-	if err != nil {
-		return managed.ExternalCreation{}, awsclient.Wrap(err, errCreateFailed)
+	if cr.Spec.ForProvider.FromBackup == nil {
+		req := e.client.CreateDBInstanceRequest(rds.GenerateCreateDBInstanceInput(meta.GetExternalName(cr), pw, &cr.Spec.ForProvider))
+		_, err = req.Send(ctx)
+		if err != nil {
```

Comment:
```suggestion
		if _, err = req.Send(ctx); err != nil {
```

Just a little style nit. Feel free to ignore, but the above suggestion is arguably more idiomatic in Go.

Reply:
Sure, thanks for the feedback - this was some copypasta from existing code but I'll update it.

Reply:
Resolved by prior changes to the code due to AWS SDK updates.